cmd/puller/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"flag"
+	"github.com/kubeflow/kfserving/pkg/puller"
+)
+
+var (
+	configDir  = flag.String("config-dir", "/mnt/configs", "directory for multi-model config files")
```

Comment:
```suggestion
	configDir  = flag.String("config-dir", "/mnt/configs", "directory for model config files")
```

According to the comments in other PRs we want to can the config model config instead of multi-model config. The rational is that we want to mount this sidecar and configmap to all the InferenceService pods so a single model InferenceService is really a syntactic sugar of MMS.